### ISSUE TITLE
The last layer is ignored

### ISSUE THREAD

**Issue by whdlgp** [hahnyuan/darknet-visualizer#2]
# Problem
* The last layer is ignored when reading cfg file and store it to options
* For example, if read [yolov4.cfg](https://github.com/AlexeyAB/darknet/blob/master/cfg/yolov4.cfg). The last "yolo" layer will be ignored

# Solution
Just add one line in [read_cfg.py](https://github.com/hahnyuan/darknet-visualizer/blob/master/read_cfg.py)

like this, in read_cfg.py
```python
        elif(i[0] in ['\0','#',';']):
            pass
        else:
            read_option(i,current)
    options.append(current.copy()) #add this after line 19
    file.close()
    return options[1:]
```

This Darknet cfg visualizer is simple and nice.
If this repository is still alive, Please check this :)

If apply this, I can read Yolo v4, Yolo v4 Tiny and Yolo v4 Tiny 3l.

**Comment by JeremyKeusters** [comment]
Thanks for sharing this! Saved me the effort of debugging. :)
I can confirm that it also works for YOLOv7-tiny visualisation.

**Comment by whdlgp** [comment]
If you mean this,
https://github.com/WongKinYiu/yolov7
There is no config file for darknet.

Maybe you can visualize it with this tool
https://github.com/lutzroeder/Netron

**Comment by JeremyKeusters** [comment]
No, I meant that it actually also does work for YOLOv7-tiny. AlexeyAB released a cfg file [here](https://github.com/AlexeyAB/darknet/issues/8595#issue-1296709732).

**Comment by whdlgp** [comment]
Yes, It works.

below image is YOLOv7-tiny graph generated by this visualizer

![Imgur](https://i.imgur.com/N8xfXXt.png)
